networking/multiple_networks/configuring-additional-network.adoc

```diff
@@ -26,7 +26,13 @@ For an additional network, IP addresses are provisioned through an IP Address Ma
 [id="{context}_configuration-additional-network-attachment"]
 == Configuration for an additional network attachment
 
-An additional network is configured via the `NetworkAttachmentDefinition` API in the `k8s.cni.cncf.io` API group. The configuration for the API is described in the following table:
+An additional network is configured via the `NetworkAttachmentDefinition` API in the `k8s.cni.cncf.io` API group.
+[IMPORTANT]
+====
+Do not store any sensitive information or a secret in the `NetworkAttachmentDefinition` object because this information is accessible by the project administration user.  
```

Comment:
I'm confused about the logic here -- it suggests that, by definition, project admins are not to be trusted with 'sensitive information'. Surely that's a call that the cluster admins should make?

I wonder if we should be saying something more like this:

"The `NetworkAttachmentDefinition` object can be accessed by both project and cluster administrators. You should not store sensitive information in this object if that information cannot be shared with all project administrators for your cluster."

What do you think? @Amrita42

Reply:
Sounds good , I have re-phrased as follows:
The NetworkAttachmentDefinition object is accessible by both project and cluster administrators. You should not store any sensitive information in this object if that information cannot be shared with the project administrators for your cluster. @ctauchen.

Reply:
let me re-phrase , thanks @ctauchen :+1:

Reply:
Super -- @zhaozhanqi does this still look good to you?

Reply:
yes, LGTM, sorry for late reply